src/frontend/providers/Currency.provider.tsx

```diff
@@ -36,13 +36,23 @@ const CurrencyProvider = ({ children }: IProps) => {
     SessionGateway.setSessionValue('currencyCode', currencyCode);
   }, []);
 
+  const currencyCodeList = currencyCodeListUnsorted.sort((n1, n2) => {
```

Comment:
I'm not sure what `currencyCodeListUnsorted` is seems just like an array of strings though. `Array.sort()` should suffice in that case.

Reply:
When I wrote the code, I felt like I was doing something unnatural... and yes `sort()` is all I needed without having to define actual compare logic.